packages/auth/src/authClients/clerk.ts

```diff
@@ -42,6 +46,42 @@ export const clerk = (client: Clerk): AuthClientClerk => {
     ) => clerkClient(client)?.signOut(callbackOrOptions as any, options),
     signup: async (options?: SignUpProps) =>
       clerkClient(client)?.openSignUp(options || {}),
+    restoreAuthState: async () => {
+      const clerk = clerkClient(client)
+      if (!clerk) {
+        // If the client is null, we can't restore state or listen for it to happen.
+        // This behavior is somewhat undefined but it should be handled by `useIsWaitingForClient.
+        // For now we'll just return.
+        return
+      }
+
+      // NOTE(zack): Clerk's API docs say session will be undefined if loading (null if loaded and confirmed unset).
```

Comment:
Could you clean this up please? We don't usually leave our names in comments in the code.

Reply:
Yup sorry - force of habit from work codebase